Source/CsDebugScript.Engine/Engine/SymbolProviders/DiaSymbolProvider.cs

```diff
@@ -243,12 +243,12 @@ public VariableCollection GetFrameLocals(StackFrame stackFrame, bool arguments)
         /// <param name="module">The module.</param>
         private ISymbolProviderModule GetDiaModule(Module module)
         {
-            if (module.SymbolProvider == null)
+            if (module == null)
             {
-                module.SymbolProvider = modules[module];
+                return null;
             }
 
-            return module.SymbolProvider;
+            return module.SymbolProvider ?? (module.SymbolProvider = modules[module]);
```

Comment:
This is not so readable. Can you please return to previous if and keep the current if?